scripts/Sensitivity/sensitivity.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python
+"""Sparger Sensitivity Analysis
+===============================
+
+This script allows the user to find the optimum designs for optimal
+sparger (bubble generator) and entrainment separator efficiency of
+molten salt breeder reactor. The script will put all output/plots
+into 'data' directory under the working directory. Results are provided
+in results.cvs file.
+
+This main script requires that \n
+'pandas'\n
+'numpy'\n
+'matplotlib'\n
+'itertools'\n
+be installed within the Python environment you are running this script in.
+
+Use following command to make sensitivity analysis,
+
+python sensitivity.py
+
+"""
+
+import sys
+sys.path.append('../../')
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from itertools import product
+from sparging import Sparger
+from sparging import Separator
+
+
+def ref_sparger_design():
```

Comment:
Ideally this should be a class that inherits from sparger and can be selected by the user when running saltproc, rather than a whole script that would have to be copy-pasted for all attempts.